projects/ccf-rui/src/environments/environment.staging.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
+  production: true,
+  googleAnalyticsToken: 'UA-136932895-2'
```

Comment:
This needs to be a copy of environment.prod.ts (I'll get you the staging GA token to change, but that'll be the only difference).